src/ComplexType.php

```diff
@@ -194,13 +184,28 @@ protected function generateClass()
             $constructorSource,
             $constructorComment
         );
-        $class->addFunction($constructor);
+        $this->class->addFunction($constructor);
 
         foreach ($accessors as $accessor) {
-            $class->addFunction($accessor);
+            $this->class->addFunction($accessor);
+        }
+    }
+
+    /**
+     * Determine parent class
+     * @return string
```

Comment:
This returns a string containing the PHP identifier for the parent class or `null` if there is no applicable parent class.